docs/UserGuide.md

```diff
@@ -125,23 +125,21 @@ Examples:
   * correct answers include: "haRRy PoTtEr", "Harry Potter and the Philosopher's Stone",
   "potter harry", "harrypotter" and "wordthatincludesharryandpotter"
   * incorrect answers: "Harry", "Potter", "harr pottery"
+  
+### List all items : `list`
```

Comment:
Can we change this to List All Items (capitalise)? To be consistent with the other headers

Reply:
Quite a number of the other headers aren't capitalised though? For example `Exit the program` or `Change the theme` are some of the cases in the UG.

I presume stuff like `True False Question` are only capitalised because they're a type of question or something? I'm not really sure about this as well.